runtime/ftplugin/karel.vim

```diff
@@ -0,0 +1,22 @@
+" Vim filetype plugin file
+" Language:    KAREL
+" Last Change: 2024-11-18
+" Maintainer:  Kirill Morozov <kirill@robotix.pro>
+" Credits:     Patrick Meiser-Knosowski for the initial implementation.
+
+if exists("b:did_ftplugin")
+  finish
+endif
+
+let b:did_ftplugin = 1
+let s:keepcpo = &cpo
+set cpo&vim
```

Comment:
you can drop this block (and the restoration at the end of the file)

Reply:
inkluding line 11? I see we don't need keepcpo since there is no line continuation, but line 11...?

Reply:
I kept line 11 and removed everything else.

Reply:
thanks. Exactly like this :)

Reply:
We don't have this very specific, only `:h write-filetype-plugin`. When I review a PR, I try to check those settings and if it looks like it isn't needed, I give the advice to leave those things out. But older runtime files may still have it.